src/utils/string/githubPRLinkify.tsx

```diff
@@ -0,0 +1,11 @@
+import ReactDOMServer from "react-dom/server";
+import { StyledLink } from "components/styles/StyledLink";
+
+export const githubPRLinkify = (unlinkified: string) =>
+  unlinkified.replace(
+    /https:\/\/github.com\/[a-zA-Z0-9-]+\/[a-zA-Z0-9-]+\/pull\/\d+/gi,
+    (match) =>
+      ReactDOMServer.renderToString(
+        <StyledLink href={match}>{match}</StyledLink>
```

Comment:
It seems a little redundant to include the entire URL both written out and as a link destination. Something like this could include the same information and be significantly less noisy:

<img width="918" alt="image" src="https://user-images.githubusercontent.com/9298431/133621530-678836dd-546d-43e9-aabb-2cf25b597937.png">

I wonder if at some point we should look into making app changes to update the `message`. Another option within Spruce could be to replace the `match` text here with some other text, like "(permalink)" or "(GitHub)" or something.

However, I think this is a great improvement for now, no need to to block merging!

Reply:
Great Idea! We should bring this up in the design sync tho since this would mean changing the commit message contents. We would also need to keep this uniform in other places we show commit messages such as the waterfall and history pages to avoid confusion. 

But i agree this might be out of scope so no need to block merging on it.